python/orca/src/bigdl/orca/learn/pytorch/core/base_ray_estimator.py

```diff
@@ -102,17 +101,24 @@ def setup(self, params, backend='ray', runner_cls=None, workers_per_node=1):
                 for i, worker in enumerate(self.remote_workers)
             ])
 
-            driver_ip = get_driver_node_ip()
-            driver_tcp_store_port = find_free_port()
-
-            _ = dist.TCPStore(driver_ip, driver_tcp_store_port, -1, True,
-                              dist.constants.default_pg_timeout)
-
-            ray.get([
-                worker.setup_torch_distribute.remote(
-                    driver_ip, driver_tcp_store_port, i, num_nodes)
-                for i, worker in enumerate(self.remote_workers)
-            ])
+            if mode == "fit":
```

Comment:
What about evaluate? Should also setup ddp for evaluate?

Reply:
Evaluate only needs DistributedSampler.

Reply:
Then setup_predict_distribute may not be a good name; just setup?